pynest/nest/tests/test_sp/test_disconnect.py

```diff
@@ -21,6 +21,9 @@
 
 import nest
 import unittest
+import numpy as np
+
+__author__ = 'naveau'
```

Comment:
Isn't this line duplicated now? Please l.28 see below.

Reply:
Done!